internal/api/front/perses/src/app/project/dashboard/dashboard-list/dashboard-list.component.html

```diff
@@ -0,0 +1,20 @@
+<app-page>
```

Comment:
Every file needs to contain the copyright

Reply:
for the moment I didn't put the copyright on the html file. I'm not sure if it's fine or not

Reply:
We don't have it in the existing html files actually ?